OrbitGl/Batcher.cpp

```diff
@@ -103,6 +103,28 @@ void Batcher::AddTriangle(const Triangle& triangle, const Color& color, const Co
   user_data_.push_back(std::move(user_data));
 }
 
+void Batcher::AddCircle(const float& z, const float& radius, const float& x_pos, const float& y_pos,
```

Comment:
Because simple types are passed in through registers, passing const ref is overkill here, we should pass in floats and even Vec2 by value.

Also, I'd prefer to reorder the parameters: AddCircle(Vec2 position, float radius, float z);

Reply:
Thank you so much! I did know know that simple types are passed in though registers.